github_api/voting.py

```diff
@@ -160,21 +161,29 @@ def parse_review_for_vote(state):
 
 def parse_reaction_for_vote(body):
     """ turns a comment reaction into a vote, if possible """
-    try:
```

Comment:
Maybe that was some underhanded python ([a la...](https://en.wikipedia.org/wiki/Underhanded_C_Contest))?